go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/kubernetes-sigs/depstat
 
 go 1.16
 
-require github.com/spf13/cobra v1.1.1
+require github.com/spf13/cobra v1.1.3
```

Comment:
We're not bumping to cobra 1.1.3 since 1.1.1 is what is being used in the kubernetes codebase and bumping it would mean adding an extra dependency :)

Reply:
1. Revert the code back to 1.1.1
1. Revert powershell method to be 1.1.1 compliant
1. Manual Functional Acceptance test completed

@RinkiyaKeDad let me know what else need to be done.

Reply:
Thanks @GrooveCS, can you please fix the failing lint check. Once that is done we can request a final review from @alenkacz 🙂